clusterloader2/testing/huge-service/modules/measurements.yaml

```diff
@@ -12,6 +12,11 @@
 {{$ENABLE_SYSTEM_POD_METRICS := DefaultParam .ENABLE_SYSTEM_POD_METRICS true}}
 {{$RESTART_COUNT_THRESHOLD_OVERRIDES := DefaultParam .RESTART_COUNT_THRESHOLD_OVERRIDES ""}}
 {{$USE_SIMPLE_LATENCY_QUERY := DefaultParam .USE_SIMPLE_LATENCY_QUERY false}}
+{{$CLUSTER_OOMS_IGNORED_PROCESSES := DefaultParam .CL2_CLUSTER_OOMS_IGNORED_PROCESSES ""}}
+{{$ENABLE_CLUSTER_OOMS_TRACKER := DefaultParam .CL2_ENABLE_CLUSTER_OOMS_TRACKER true}}
```

Comment:
Shouldn't this have equal value to `ENABLE_CONTAINER_RESTARTS_MEASUREMENT`?

Reply:
This parameter is also programatically false by default: https://github.com/kubernetes/perf-tests/blob/96d5b0a3cccb7475046946d2dc7c9adef3970f0d/clusterloader2/pkg/measurement/common/ooms_tracker.go#L88